docs/evaluation/examples.rst

```diff
@@ -169,10 +169,12 @@ interrogate the object.
 ::
 
    >>> from sherpa.astro.io import read_pha
-   >>> pha = read_pha('9774.pi')
-   read ARF file 9774.arf
-   read RMF file 9774.rmf
-   read background file 9774_bg.pi
+   >>> pha = read_pha('sherpa-test-data/sherpatest/9774.pi')
```

Comment:
Having to explicitly set the path here is only really useful for the test and doesn't really help the user other than making the text longer. Is there any way to add code which is only used by doctest so we could start this section with something like

    import os
    thisdir = os.getcwd()
    os.chdir("sherpa-test-data/sherpatest")

and then at the end

    os.chdir(thisdir)

? I imaging this isn't going to be the only place I'd want to see this.

Reply:
This might be a good case for a (not visible) setup.

Reply:
I don't think we need to resolve this for this PR, but would like it looked into in the glorious future when we have time to do so.